ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -5,4 +5,20 @@ import DynamicQuantities: UnionAbstractQuantity, ustrip, dimension, new_quantity
 
 norm(q::UnionAbstractQuantity, p::Real=2) = new_quantity(typeof(q), norm(ustrip(q), p), dimension(q))
 
+# Define isapprox for vectors of Quantity types
+function Base.isapprox(
+    u::AbstractArray{Q}, v::AbstractArray{Q};
```

Comment:
Will promotion rules automatically run beforehand? Otherwise should add another method for `Q1` and `Q2`. Also for `QuantityArray` and `AbstractArray`

Reply:
I've updated this section to include versions specialized for `QuantityArray` and `AbstractArray` allowing for type-inequality between the arguments. This commit also added unit tests to exercise both of these method definitions, and all tests currently pass.